.travis.yml

```diff
@@ -16,8 +16,6 @@ env:
   - TEST_SUITE=vmdb PARALLEL=true
   - TEST_SUITE=brakeman
 matrix:
-  allow_failures:
-  - rvm: 2.5.3
   exclude:
   - rvm: 2.5.3
     env: TEST_SUITE=brakeman
```

Comment:
Should this be removed too?

Reply:
Ah good catch. Also I wan't sure if we wanted
```
  exclude:
  - rvm: 2.5.3
    env: TEST_SUITE=brakeman
```

I was going to ask you and @jrafanie

Reply:
Maybe we should remove it completely and see if brakeman on 2.5 is a problem.  I don't remember if we excluded this because running brakeman on more than one ruby was slow or something else.